src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartTokenFormatterFormatRangeTests.cs

```diff
@@ -446,12 +446,10 @@ class Class
 class Class
 {
     int Prop {
-        get
-        {
-            return 1;
+        get { return 1;
         }";
 
-            await AutoFormatOnCloseBraceAsync(code, expected, SyntaxKind.GetKeyword);
+            await AutoFormatOnCloseBraceAsync(code, expected, SyntaxKind.OpenBraceToken);
```

Comment:
❗️ Do we know why this change occurred? I actually use this sequence of operations relatively frequently to convert a single-line getter into a multi-line getter.

Reply:
➡️ It turns out that this behavior change is due to the fact that the property declaration does not have a closing brace. In the use case I was primarily worried about, this does not occur. I added a new test showing that the scenario I was worried about has not been altered by this change.